extensions/pkg/controller/utils.go

```diff
@@ -119,6 +119,9 @@ var EnsureFinalizer = controllerutils.EnsureFinalizer
 // If it is set, it removes it and issues an update.
 var DeleteFinalizer = controllerutils.RemoveFinalizer
 
+// PatchDeleteFinalizer ensures that the given finalizer is not present anymore in the given object via a patch request.
+var PatchDeleteFinalizer = controllerutils.PatchRemoveFinalizers
```

Comment:
Can you avoid this alias?
Last time when I was looking how to remove a finalizer, I found 2 different implementations in g/g and one alias which which is misleading to me.